tests/slt/rowsort.slt

```diff
@@ -1,6 +1,23 @@
+control resultmode valuewise
+
+query III rowsort
+select * from example_sort
+----
+1
+10
+2333
+10
+100
+2333
+2
+20
+2333
+
+control resultmode rowwise
+
 query III rowsort
 select * from example_sort
 ----
 1 10 2333
 10 100 2333
-2 20 2333
+2 20 2333
```

Comment:
could you please also help add a test case for `valuesort`? (might require some modifications for `FakeDB`)